example_3/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2_control_demo_example_3</name>
+  <version>0.0.0</version>
+  <description>Demo package of `ros2_control` hardware for RRbot with multi interface.</description>
+
+  <maintainer email="denis@stogl.de">Denis Štogl</maintainer>
```

Comment:
Can I join the club of maintainers here please? ;)

Reply:
ok, I can do that all at once after the examples are merged